docs/analysts/development/gems/source-target/file/excel.md

```diff
@@ -21,3 +21,29 @@ You can also use the [upload file](docs/analysts/development/gems/source-target/
 | Location type               | Location you want to connect from.                                |
 | Select or create connection | Whether to select an existing connection, or to create a new one. |
 | Filepath                    | The file path according to the connection type.                   |
+
+### Source properties
+
+| Property                      | Description                                            | Default  |
+| ----------------------------- | ------------------------------------------------------ | -------- |
+| Description                   | Description of the table.                              | None     |
+| Header                        | Whether the first row contains column headers.         | Enabled  |
+| Allow Undefined Rows          | Allows rows with all values undefined (null or empty). | Enabled  |
+| Allow Incomplete Rows         | Allows rows with missing values for some columns.      | Enabled  |
+| Ignore Cell Formatting        | Ignores Excel-specific formatting during data parsing. | Enabled  |
```

Comment:
Ignore Cell Formatting specifies whether to apply the number format for the cell value or get the raw value.